code/modules/reagents/chemistry/reagents/unique/eigenstasium.dm

```diff
@@ -0,0 +1,250 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////
+//										EIGENSTASIUM
+///////////////////////////////////////////////////////////////////////////////////////////////////
+//eigenstate Chem
+//Teleports you to chemistry and back
```

Comment:
As a default effect this seems kinda... ahhhh corny? to me. I'm not sure I can say exactly why, but it's always struck me as a bit contrived. 

Is this behavior you're interested in? @Rohesie

Reply:
Well it's not just to chemsitry, it's where you react it. I'll change to comments to reflect that!

Reply:
I don't have strong opinions, @LemonInTheDark 
What worries you? Is it a balance issue? Something else? Or do you have a different behavior in mind you think would be better?

Reply:
It just seems kinda out of scope for chemistry, feels like someone pring a new xenobio strain with a lot of unrelated features. IDK